### PR TITLE
docs(upgrade): upgrade from 4.3 to 4.4

### PR DESCRIPTION
**Pull request by zhongwencool**

### CODE REVIEW

File: zh_CN/changes/upgrade-4.4.md [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+
+# 升级到 4.4 版本
+
+以下内容仅针对从 4.3 升级到 4.4 的用户。
+
+4.3和4.4 的节点可以运行在同一集群中，可以采用滚动升级从4.3 升级到 4.4 。
+
+推荐按以下步骤完成滚动升级：
+
+1. 在 4.2 节点上备份data及etc目录（见下文）。
```
Comment (zmstone) [inline zh_CN/changes/upgrade-4.4.md]: 4.3

File: zh_CN/changes/upgrade-4.4.md [hunk 2]
```diff
@@ -0,0 +1,107 @@
...
+* 环境变量 `EMQX_NODE__DATA_DIR` 指向的位置
+* 配置文件中 `node.data_dir` 指向的位置
+* 在docker中运行的默认位置: `/opt/emqx/data` `opt/emqx/etc` （通常是一个挂在的外部volume）
+* 直接使用 zip 安装包的默认位置: `<install-path>/data` `<install-path>/etc`
+* 使用RPM或DEB安装包安装的默认位置：`/var/lib/emqx/` `/etc/emqx`
+
+以RPM默认安装为例，你需要
+```bash
+
+// 创建备份文件夹
```
Comment (zmstone) [inline zh_CN/changes/upgrade-4.4.md]:
```suggestion
## 创建备份文件夹
```

File: zh_CN/changes/upgrade-4.4.md [hunk 3]
```diff
@@ -0,0 +1,107 @@
...
+mkdir -p ～/emqx-backup/etc/
+mkdir -p ～/emqx-backup/data/
+
+// 确认本节点emqx已经停止运行
```
Comment (zmstone) [inline zh_CN/changes/upgrade-4.4.md]:
```suggestion
## 确认本节点emqx已经停止运行
```

File: zh_CN/changes/upgrade-4.4.md [hunk 4]
```diff
@@ -0,0 +1,107 @@
...
+systemctl stop emqx
+systemctl status emqx 
+
+//复制备份文件
```
Comment (zmstone) [inline zh_CN/changes/upgrade-4.4.md]:
```suggestion
## 复制备份文件
```

File: zh_CN/changes/upgrade-4.4.md [hunk 5]
```diff
@@ -0,0 +1,107 @@
...
+cp -r /var/lib/emqx/ ～/emqx-backup/data/
+
+
+```
+
+## 卸载 4.3 版本
+
+以RPM默认安装为例：
+```bash
+//查看确认安装的版本是否为需要卸载的
```
Comment (zmstone) [inline zh_CN/changes/upgrade-4.4.md]:
```suggestion
## 查看确认安装的版本是否为需要卸载的
```

File: zh_CN/changes/upgrade-4.4.md [hunk 6]
```diff
@@ -0,0 +1,107 @@
...
+rpm -qa |grep emqx
+
+//确认无误后卸载旧版本
```
Comment (zmstone) [inline zh_CN/changes/upgrade-4.4.md]:
```suggestion
## 确认无误后卸载旧版本 
```

File: zh_CN/changes/upgrade-4.4.md [hunk 7]
```diff
@@ -0,0 +1,107 @@
...
+systemctl status emqx
+```
+- 通过emqx_ctl 查看集群状态
+```bash
+/usr/bin/emqx_ctl cluster status
+
+```
+- 查看日志是否有异常
+```bash
+// 找到最新的写入日志文件
```
Comment (zmstone) [inline zh_CN/changes/upgrade-4.4.md]:
```suggestion
## 找到最新的写入日志文件
```

File: zh_CN/changes/upgrade-4.4.md [hunk 8]
```diff
@@ -0,0 +1,107 @@
...
+ls -htl /var/log/emqx/emqx.log.*[0-9] |head -n 1
+// 查看此日志：
```
Comment (zmstone) [inline zh_CN/changes/upgrade-4.4.md]:
```suggestion
## 查看此日志：
```

File: en_US/changes/upgrade-4.4.md [hunk 9]
```diff
@@ -0,0 +1,104 @@
+# Upgrade from 4.3 to 4.4
+
+EMQ X 4.3 and 4.4 nodes co-exist in the same cluster.
+Hence it's possible to apply a rolling upgrade of a EMQ X cluster.
+
+The upgrade steps of each EMQ X node is summarised as below:
```
Comment (k32) [inline en_US/changes/upgrade-4.4.md]:
```suggestion
The upgrade steps of each EMQ X node are summarised below:
```

File: en_US/changes/upgrade-4.4.md [hunk 10]
```diff
@@ -0,0 +1,104 @@
...
+
+1. Optional: remove the node from load-balancer
+1. Stop the node, (e.g. `emqx sotp`, `systemctl stop emqx`)
```
Comment (k32) [inline en_US/changes/upgrade-4.4.md]:
```suggestion
1. Stop the node, (e.g. `emqx stop`, `systemctl stop emqx`)
```

File: en_US/changes/upgrade-4.4.md [hunk 11]
```diff
@@ -0,0 +1,104 @@
...
+1. Backup 'data' and 'etc' directories. (more detila below)
```
Comment (k32) [inline en_US/changes/upgrade-4.4.md]:
```suggestion
1. Backup 'data' and 'etc' directories. (more detail below)
```

**Review by k32 — APPROVED** [review]

File: en_US/changes/upgrade-4.4.md [hunk 12]
```diff
@@ -0,0 +1,104 @@
...
+
+1. Optional: remove the node from load-balancer
+1. Stop the node, (e.g. `emqx stop`, `systemctl stop emqx`)
+1. Backup 'data' and 'etc' directories. (more details below)
+1. Uninstall v4.3
+1. Install v4.4, and restore the backup directories for 'data' and 'etc'
+1. Start v4.4, and monitor alerts/error logs
+1. Optional: add the node back to the load-balancer
+
+NOTE: Mind the [Config changes](#config-change) and [Behaviour changes](#behaviour-changes).
```
Comment (thalesmg) [inline en_US/changes/upgrade-4.4.md:16]: Nit: these links are not working (at least in the Github visualization)

File: en_US/changes/upgrade-4.4.md [hunk 13]
```diff
@@ -0,0 +1,104 @@
...
+```
+
+## Start v4.4
+
+- Taking systemctl for example
+
+```bash
+# Start
+systemctl start emqx
+# Innspect status
```
Comment (thalesmg) [inline en_US/changes/upgrade-4.4.md:80]: Nit: _Inspect_, or maybe _Check_

File: en_US/changes/upgrade-4.4.md (hunk 12 above)
Comment (zmstone) [reply en_US/changes/upgrade-4.4.md:16]: it's been fixed in the next PR.

**Pull request merged**